app/build.gradle

```diff
@@ -48,6 +52,12 @@ dependencies {
     // 参考 https://qiita.com/ymmtyuhei/items/2cb79031e20e1dcff6fc
     testImplementation 'com.nhaarman.mockitokotlin2:mockito-kotlin:2.0.0-RC1'
     testImplementation 'org.mockito:mockito-android:2.22.0'
+
+    androidTestImplementation 'androidx.test:rules:1.1.0'
+    // Optional -- Hamcrest library
+    androidTestImplementation 'org.hamcrest:hamcrest-library:1.3'
+    // Optional -- UI testing with Espresso
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.0'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/aiphone-test-muramatsu/ChatApp/blob/3c2510bdb7b7beb3be599dd8fe41546d6da1decb/app/build.gradle#L60"></span>A newer version of androidx.test.espresso:espresso-core than 3.1.0 is available: 3.2.0</del></td>
    </tr>
  </tbody>
</table>